components/common/logo/Logo.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+
+var curenetics_logo = require('./curenetics_logo_2.png');
+
+const Logo = styled.img`
+  height: 45px;
+  width: 115px;
+  overflow: hidden;
+  top: 20px;
+  left: 50px;
+  position: absolute;
+`;
+
+class Logo extends Component {
+  render() {
```

Comment:
Make this a functional component, don't need a class.
Better yet, combine it into a Header / TopBar.